### PR TITLE
removed unused code in alphaWeight.glsl, fixed curly braces to follow…

### LINKED ISSUE #5727: Clean up czm_alphaWeight
via @pmconne 

In `czm_alphaWeight` we compute the fragment position in eye space when it isn't needed. It was used in previous versions, but can be removed.

### PR DESCRIPTION
**Pull request by loshjawrence**
Fixes #5727 
removed unused code, fixed curly braces to follow code style

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@loshjawrence, thanks for the pull request! Maintainers, we have a signed CLA from @loshjawrence, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by lilleyse** [comment]
@loshjawrence - fine for this PR, but in the future it is best to create a new branch name for each PR.

File: Source/Shaders/Builtin/Functions/alphaWeight.glsl [hunk 1]
```diff
@@ -1,32 +1,10 @@
 /**
  * @private
  */
-float czm_alphaWeight(float a)
-{
-    float x = 2.0 * (gl_FragCoord.x - czm_viewport.x) / czm_viewport.z - 1.0;
-    float y = 2.0 * (gl_FragCoord.y - czm_viewport.y) / czm_viewport.w - 1.0;
+float czm_alphaWeight(float a) {
```
Comment (lilleyse) [inline Source/Shaders/Builtin/Functions/alphaWeight.glsl]: Our GLSL code style differs from our JS style, and we actually leave the curly brace on its own line.

**Comment by lilleyse** [comment]
The rest of the changes look good!

**Comment by lilleyse** [comment]
All looks good. Thanks.

Another small point - whenever code is ready to be looked at again just leave a short comment like "Updated" so that the reviewer gets notified.

**Pull request merged**